File: Examples/CSharp/Articles/HtmlExportFrameScripts.cs
using System;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    class HtmlExportFrameScripts
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Open the required workbook to convert
            Workbook wb = new Workbook(sourceDir + "sampleHtmlExportFrameScripts.xlsx");

            // Disable exporting frame scripts and document properties
            HtmlSaveOptions options = new HtmlSaveOptions();
            options.ExportFrameScriptsAndProperties = false;

            // Save workbook as HTML
            wb.Save(outputDir + "outputHtmlExportFrameScripts.html", options);

            Console.WriteLine("HtmlExportFrameScripts executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/HtmlExportFrameScripts.vb
Imports Aspose.Cells

Namespace Articles
    Public Class HtmlExportFrameScripts
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Open the required workbook to convert
            Dim w As Workbook = New Workbook(dataDir + "Sample1.xlsx")

            ' Disable exporting frame scripts and document properties
            Dim options As HtmlSaveOptions = New HtmlSaveOptions()
            options.ExportFrameScriptsAndProperties = False

            ' Save workbook as HTML
            w.Save(dataDir + "output.out.html", options)

            Console.WriteLine("Press any key to continue...")
            Console.ReadKey()
            ' ExEnd:1

        End Sub
    End Class
End Namespace
